flink-cdc-base/src/main/java/com/ververica/cdc/connectors/base/source/enumerator/IncrementalSourceEnumerator.java

```diff
@@ -197,6 +233,29 @@ private void syncWithReaders(int[] subtaskIds, Throwable t) {
                         subtaskId, new FinishedSnapshotSplitsRequestEvent());
             }
         }
+
+        suspendBinlogReaderIfNeed();
+        wakeupBinlogReaderIfNeed();
```

Comment:
Same as comment above.

Reply:
Adressed in https://github.com/ververica/flink-cdc-connectors/pull/1838/commits/7490d9b9785adb1db065b68cd2305458e46920aa